packages/docs/blog/2023-05-15-unirep-in-turkey.md

```diff
@@ -0,0 +1,34 @@
+---
+slug: unirep-travels-to-turkey-2023
+title: UniRep Travels to Turkey, 2023
+authors: [anthony]
+tags: [unirep in turkey]
+---
+
+<img src="static/img/../../../static/img/engin-yapici-WA1u0scVLZU-unsplash.jpg" alt="Alt Text" width="600" style="display: block; margin: 0 auto;">
```

Comment:
you missed a `/` before `>`
and I think the path could be `../static/img/engin-yapici-WA1u0scVLZU-unsplash.jpg`